.env

```
@@ -0,0 +1,7 @@
+VITE_API_KEY=AIzaSyDE0ejz8Du4Qe4XXW4lJqyofIveuUz6kC0
+VITE_AUTH_DOMAIN=finlit-dc8ca.firebaseapp.com
+VITE_PROJECT_ID=finlit-dc8ca
+VITE_STORAGE_BUCKET=finlit-dc8ca.firebasestorage.app
+VITE_MESSAGING_SENDER_ID=932672415600
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_MESSAGING_SENDER_ID key should go before the VITE_PROJECT_ID key

(UnorderedKey)